lua/telescope/actions/set.lua

```diff
@@ -13,7 +13,7 @@
 local a = vim.api
 
 local log = require('telescope.log')
-local path = require('telescope.path')
+local path = require('plenary.path')
```

Comment:
Maybe we should make it more consistant. Sometimes `path` is written starting with a big letter and sometimes not.

If i have to choose i would take `Path`

Reply:
I agree - I had just tried to leave things as they were, but `Path` seems better to me.